openstax-poland/adaptarr-front
# Processes routing

Routing for processes from https://github.com/openstax-poland/adaptarr-backlog/issues/44

### Testing
1. Go to Editing Processes
2. Click on preview icon
3. Url should change and preview should be opened
4. Refresh - everything should stay the same
5. Click close preview
6. url should change
7. Click edit icon
8. Url should change
9. Refresh - everything should stay the same
10. Click Cancel
11. Url should change
12. Click create new
13. url should change
14. Refresh - everything should stay the same
15. Change url to something like `/processes/9999/preview` and hit enter
16. There should be an error message.
